src/Worker.ts

```diff
@@ -0,0 +1,37 @@
+import { ref, onMounted, onUnmounted, Ref } from '@vue/composition-api';
+
+export function useWorker(url: string) {
+  let message: Ref<any> = ref(null); 
+  let ww: Worker;
```

Comment:
Could we just rename this to `worker` I don't think it is necessary to have `webWorker` or abbreviations of it since its the only function that it will be referenced in.

Reply:
Sure, can fix that